relay-server/src/actors/project.rs

```diff
@@ -706,6 +758,17 @@ impl Project {
         }
     }
 
+    fn set_state(&mut self, state: Arc<ProjectState>, aggregator: Addr<Aggregator>) {
+        let buckets = self.state.set_state(state);
+
```

Comment:
We should drop buckets here if the project has been disabled, see

https://github.com/getsentry/relay/blob/a00778f110e818c6ce3a5c8e381a05466f04a26d/relay-server/src/actors/project.rs#L899-L902